stablehlo/reference/Ops.cpp

```diff
@@ -519,9 +526,8 @@ SmallVector<InterpreterValue> eval(Region &region,
       auto rhs = scope.findTensor(op.getRhs());
       auto rank = lhs.getRank();
 
-      SmallVector<int64_t> windowStrides(rank - 2, 1);
-      if (auto windowStridesAttr = op.getWindowStrides())
-        windowStrides = SmallVector<int64_t>(windowStridesAttr.value());
+      SmallVector<int64_t> windowStrides = extractAttributeOrDefault<int64_t>(
+          op.getWindowStrides(), rank - 2, 1);
```

Comment:
`windowReversal` is another one that is affected by this, and these variables are being passed around multiple times, so it's better to keep them as `SmallVector`. I propose to keep the current formulation for consistency across different variables (since we custom handle this via `extractAttributeOrDefault`, it can be written in 1 line too).

Reply:
Gotcha. I mostly proposed this because I find `extractAttributeOrDefault` kinda hard to understand without looking at the implementation. (For me, this isn't really about code size, just trying to make the logic as straightforward as possible.) Especially, the fact that the last two arguments are going to be fed directly into the constructor for SmallVector is not obvious.

IMO the following would be clearer, if slightly more verbose:

```
auto windowStrides = op.getWindowStrides() ? llvm::to_vector(op.getWindowStrides.value()) : SmallVector<int64_t>(rank - 2, 1);
```

Anyway, this is a minor thing, feel free to keep the current code.